docs: Explain how to use images via binary-assets (#171)
cultureamp/kaizen-design-system: packages/binary-assets/README.md

asset-url("some/blob.png") // -> "https://<origin>/some/blob.png"
```

## If you want to use an image

> 1. Make sure the image exists in the [`kaizen-design-system-assets` repo](https://github.com/cultureamp/kaizen-design-system-assets). If it doesn't, make a PR to add it (ask the Design Systems team for review). Images in this repo will be automatically uploaded to CloudFront and will have a hashed URL to access them.

Reviewer: ```suggestion
1. Make sure the image exists in the [`kaizen-design-system-assets` repo](https://github.com/cultureamp/kaizen-design-system-assets). If it doesn't, make a PR to add it (ask the Design Systems team for review). Images in this repo will be automatically uploaded and made available via a CloudFront distribution.
```

> 2. Make sure that the `@cultureamp/kaizen-binary-assets` package is in your repo's `package.json`. This package provides you the `assetUrl` function that translates the path to the file in the `kaizen-design-system-assets` repo into its hashed CloudFront equivalent.

Reviewer: ```suggestion
2. Make sure that the `@cultureamp/kaizen-binary-assets` package is in your repo's `package.json`. This package provides you the `assetUrl` function that translates the path to the file in the `kaizen-design-system-assets` repo into its equivalent CloudFront URL.
```
